iCubGenova09/calibrators/left_leg-calib.xml

```diff
@@ -18,15 +18,19 @@
   <group name="CALIBRATION">
     <param name="calibrationType">                    12              12          12          12           12         12           </param> <!--
     <param name="calibration1">                       24444         29251        18367       -46782     -13812      -47721     </param> -->
-    <param name="calibration1">                       40461         29421        50502       -14794     13583      -34276     </param> 
+    <param name="calibration1">                       40461         29421        50502       -13781     15352      -34276     </param> 
 
     <param name="calibration2">                       0.0            0.0        0.0           0.0         0.0       0.0     </param> 
     <param name="calibration3">                        0.0            0.0        0.0          0.0         0.0       0.0    </param> 
 
     <param name="calibration4">                       0.0            0.0        0.0          0.0         0.0       0.0      </param>
     <param name="calibration5">                       0.0            0.0        0.0          0.0         0.0       0.0      </param>
     <param name="calibrationZero">                    0.0            0.0        0.0          0.0         0.0       0.0      </param>
+<<<<<<< HEAD
     <param name="calibrationDelta">     2.657     -4.738      1.254     -0.069     15.540     -2.813</param>
+=======
+    <param name="calibrationDelta">     -2.53     -4.738      1.254     0.0     0.0     -2.813</param>
+>>>>>>> 69308424b (added fingertip support - mods for legs after rework)
```

Comment:
I think there were issues with the rebase. Since the deltas change every time, I think either versions are acceptable.

Reply:
I kept the 0 delta on the ankle pitch since I had to recalibrate the AMO/joint after rework